src/axom/quest/Shaper.cpp

```diff
@@ -135,29 +251,62 @@ void Shaper::loadShapeInternal(const klee::Shape& shape,
   revolvedVolume = discreteShape.getRevolvedVolume();
 }
 
+bool Shaper::verifyInputMesh(std::string& whyBad) const
+{
+  bool rval = true;
+
+#if defined(AXOM_USE_CONDUIT)
+  if(m_bpGrp != nullptr)
+  {
+    conduit::Node info;
+    rval = conduit::blueprint::mesh::verify(m_bpNodeInt, info);
```

Comment:
Better check that the mesh::verify() function is only used if the memory is in the right place. I think this will crash if the Conduit node contains data destined for processing in CUDA or HIP.